### PR TITLE
setup amun deployment in ocp

### PR DESCRIPTION
**Pull request by harshad16**
setup basic for amun namespaces in ocp
Signed-off-by: Harshad Reddy Nalla <hnalla@redhat.com>

## Related Issues and Dependencies

#70 

## This introduces a breaking change

- [ ] Yes
- [x] No

### CODE REVIEW

File: core/overlays/amun-api-stage/thoth-notification.yaml [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+        - name: chat-notification
+          image: registry.access.redhat.com/ubi8/ubi
+          command:
+            - "curl"
+            - "-X"
+            - "POST"
+            - "-H"
+            - "Content-Type: application/json; charset=UTF-8"
+            - "-d"
+            - "{'text':'I have successfully synchronized *Thoth Core* components to *STAGE* *amun-api* namespace, see <https://argocd-server-aicoe-argocd.apps.ocp.prod.psi.redhat.com/applications/stage-thoth-core-amun-api|ArgoCD UI> 🚀'}"
```
Comment (goern) [inline core/overlays/amun-api-stage/thoth-notification.yaml:23]: this is amun

Comment (goern) [reply core/overlays/amun-api-stage/thoth-notification.yaml:23]: *Thoth Core* should be *Amun API*

Comment (harshad16) [reply core/overlays/amun-api-stage/thoth-notification.yaml:23]: isnt this correct we want core component to sync stage amun-api namespace?

Comment (fridex) [reply core/overlays/amun-api-stage/thoth-notification.yaml:23]: I think Amun API is not part of "Thoth Core" but is a standalone "application".

Comment (harshad16) [reply core/overlays/amun-api-stage/thoth-notification.yaml:23]: oh yeah i should have made this standalone

**Comment by goern** [comment]
ping

**Comment by harshad16** [comment]
@goern  PTAL, i have moved around some of the files, 
and worked on getting the roles based on requirements, please can you review the [kustomize config](https://github.com/thoth-station/thoth-application/blob/9322f50b3d0ab5dfce5110552d74f4d6ff02c16d/amun/overlays/stage/kustomization.yaml) for stage .

**Pull request merged**